.github/ISSUE_TEMPLATE/new_idea_proposal.yml

```
@@ -0,0 +1,30 @@
+---
+name: New idea or Proposal 
+description: Suggest an idea or submit a new proposal to improvise the Ansible Community.
```

```suggestion
description: Suggest an idea or submit a new proposal to improvise the Ansible Community.
```
Can we replace the word `improvise` with something?

---

I think "help improve" should replace improvise here. Because we do not need tickets for improvisation, we already do that in the meetings without any additional instructions ;)